### DISCUSSION TITLE
Main differences between community and pro

### DISCUSSION THREAD

**Discussion by aronmolnar** [Syslifters/sysreptor#238] [Announcements]
SysReptor Community should serve freelancers and small companies as the go-to tool for report writing.  
It allows up to three users, but has no user permission: All users can read/write all reports.

SysReptor Pro has as many users as you license (50€/user/month).  
It comes with user permissions:
* Pentesters only see projects they're assigned to (unless they are superusers and elevate/"sudo" their privileges)
* There are permissions for editing finding templates, designs, users
* Guests can be further restricted

Pro comes with additional features that make pentest reporting even easier.
* Version history (see and compare changes of your report contents)
* Collaborative editing (in Community, one pentester can edit a note at a time; same upcoming for findings and report sections)
* Spell check (using languagetool)
* Multiple API keys + expiration time (in Community, a user can only have one API key and without expiration time)
* Single Sign-On (e.g., Entra ID, Google, Keycloak, etc.)
* Project archiving (encryption + offline decryption optionally with hardware tokens and four-eyes principle)
* Project retention (deletion after a certain period of time)
* Backups via API, easy to restore

We prioritize the implementation of additional features for SysReptor Pro, but Community will still benefit.  
The following feature of our roadmap will be available in CE, but limited:
* Commenting on text in markdown fields
* Document sharing via SysReptor

Features in reptor CLI will probably not be limited to Pro.

**Comment by aronmolnar** [comment]
We currently offer that if you license ten or more users at a time, we create your pentest report design for free.  
If you license less than ten users, we do it for a small allowance.

**Reply by unixerius** [reply]
I'm really happy with the quality of SysReptor and with the responsiveness on your team here on Github. I'm only a self-employed security-person, but I still shelled out for at least one license. 

Good software deserves to be paid for.

**Reply by aronmolnar** [reply]
Thank you, we appreciate this :)
